firrtl/src/main/scala/firrtl/options/Stage.scala

```diff
@@ -47,15 +49,32 @@ abstract class Stage extends Phase {
     }
   }
 
+  /** Execute this stage on some input annotations. Annotations will be read from any input annotation files.
+    * @param annotations input annotations
+    * @return output annotations
+    * @throws firrtl.options.OptionsException if command line or annotation validation fails
+    */
+  final def transform(annotations: Seq[Annotation]): Seq[Annotation] =
+    transform(seqToAnnoSeq(annotations)).toSeq
```

Comment:
For this and the `execute` method, would it be better to have the deprecated method call this one as opposed to having this call the deprecated method? Changing it this way will make the future deletion simpler.